app/scripts/lib/multichain-api/scope/supported.ts

```diff
@@ -80,5 +80,7 @@ export const isSupportedAccount = (
 // TODO: Needs to go into a capabilties/routing controller
 // TODO: These make no sense in a multichain world. accountsChange becomes authorization/permissionChanged?
 export const isSupportedNotification = (notification: string): boolean => {
-  return ['accountsChanged', 'chainChanged'].includes(notification);
+  return ['accountsChanged', 'chainChanged', 'eth_subscription'].includes(
```

Comment:
I think we may not want either `accountsChanged` but definitely not `chainChanged` supported over this API

Reply:
the CAIP specs should be updated to not show these notifications as examples anymore

Reply:
please update the spec for this function when you get a chance 🙏